Frontend/app/src/scripts/askNotify.ts

```diff
@@ -6,6 +6,6 @@ export const askNotification = (): void => {
     Notification.permission === "default"
   ) {
     //Request permission
-    Notification.requestPermission();
+    window.Notification.requestPermission();
```

Comment:
No need for window.Notification just check for if notification API is present on user browser
```
if("Notification" in window){
//INotification API in browser
}
```